up.go

```diff
@@ -295,22 +297,18 @@ func (e *Editor) String() string { return string(e.value) }
 
 func (e *Editor) DrawTo(region Region, style tcell.Style, setcursor func(x, y int)) {
 	// Draw prompt & the edited value - use white letters on blue background
-	for i, ch := range e.prompt {
-		region.SetCell(i, 0, style, ch)
-	}
-	for i, ch := range e.value {
-		region.SetCell(len(e.prompt)+i, 0, style, ch)
-	}
+	drawText(region, 0, style, string(e.prompt))
```

Comment:
If you could modify drawText to return the width of the string passed as the last parameter, we could save the widths of this and the next line in variables, and reuse them later, replacing some of the StringWidth calls:

```
wprompt := drawText(region, 0, style, string(e.prompt))
wvalue := drawText(region, wprompt, style, string(e.value))
...
```

Reply:
I also think your way better. Thanks.

Could you review 243eb17 ?
I modified drawText function and use the return value.